latex.c

```diff
@@ -813,6 +813,8 @@ void print_latex_node(GString *out, node *n, scratch_pad *scratch) {
 			g_string_append_printf(out, "\\end{description}");
 			scratch->padded = 0;
 			break;
+        case ADMONITION:
```

Comment:
Looks like I need to fix some spacing here to maintain consistency.  I will do that in an update to this PR